crossing-training.lic

```diff
@@ -656,6 +665,19 @@ class CrossingTraining
     end
   end
 
+  def climb_practice
+    check_osrel unless @researching
```

Comment:
Most of this logic appears to be a duplicate of `practice_branch`. How much can be extracted into a common method?